### ISSUE TITLE
Error with GET single transaction details ElectrumX call

### ISSUE THREAD

**Issue by zh** [Permissionless-Software-Foundation/bch-api#25]
* Problem:

When trying to get single transaction details (ElectrumX sources example - _src/routes/v3/electrumx.js_, line 373 ), instead of transaction details, there is an error message about parameter not be a string, even it is:

```
$ curl -X GET "https://api.fullstack.cash/v3/electrumx/tx/data/a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d" -H "accept: application/json"
{"success":false,"error":"txid must be a string"}
```

The POST call from the same integration examples works without problems.

**Comment by christroutner** [comment]
Hmm.. ya. I played around with it. That looks like a bug. I just realized you are using the default example:
https://github.com/Permissionless-Software-Foundation/bch-api/blob/master/src/routes/v3/electrumx.js#L373

**Comment by christroutner** [comment]
This issue fixed with #27
